developer-tools.md

```diff
@@ -216,25 +216,45 @@ In case of a failure the POD logs (driver and executors) can be found at the end
 
 Kubernetes, and more importantly, minikube have rapid release cycles, and point releases have been found to be buggy and/or break older and existing functionality.  If you are having trouble getting tests to pass on Jenkins, but locally things work, don't hesitate to file a Jira issue.
 
-<h3>Running tests in your forked repository using GitHub Actions</h3>
+<h3>Testing with GitHub Actions workflow</h3>
 
-GitHub Actions is a functionality within GitHub that enables continuous integration and a wide range of automation.
-We already have started using some action scripts and one of them is to run tests for [pull requests](https://spark.apache.org/contributing.html).
-If you are planning to create a new pull request, it is important to check if tests can pass on your branch before creating a pull request.
-This is because our GitHub Acrions script automatically runs tests for your pull request/following commits and
-this can burden our limited resources of GitHub Actions.
+Apache Spark leverages GitHub Actions that enables continuous integration and a wide range of automation. Apache Spark repository provides several GitHub Actions workflows for developers to run before creating a pull request.
```

Comment:
enables -> enable?

Reply:
I believe "GitHub Actions" is considered as an uncountable noun (?).

Reply:
e.g.) https://github.com/features/actions
> GitHub Actions makes

Reply:
I think it simply functions as singular despite "Actions". Pedantically "Github Actions, which enables" would make that smoother but it doesn't matter.